R/kendrickMass.R

```diff
@@ -0,0 +1,106 @@
+#' @title Calculate Kendrick masses from exact masses
```

Comment:
I think it would be helpful to *bundle* all these functions in one documentation *Working with Kendrick mass defects*. Basically, for one function you add the documentation for all, and use `@rdname calculateKendrickMass` for all functions. I think that would be helpful for the user as he/she does not have to jump between documentations to see what other related functions there might be.

```r
@description

The functions on this help page are related to Kendrick masses. A Kendrick mass is ...

- `calculateKendrickMass`: ...
- `calculateKendrickMassDefect`: ...


```